R/html_resources.R

```diff
@@ -85,6 +85,14 @@ find_external_resources <- function(input_file,
         discover_r_resources(file.path(input_dir, path), 
                              discover_single_resource)
       }
+      # if this is an implicitly discovered resource, it needs to refer to 
+      # a file rather than a directory 
+      if (!explicit) {
+        path_info <- file.info(file.path(input_dir, path))
```

Comment:
Hah, I'd forgotten we ported `dir_exists` to this package! That's clearer; replaced.